app/src/controllers/itemsHistory/selectors.js

```diff
@@ -0,0 +1,18 @@
+import { itemsSelector } from 'controllers/testItem';
+
+const domainSelector = (state) => state.itemsHistory || {};
+export const itemsHistorySelector = (state) => domainSelector(state).items;
+export const historySelector = (state) => domainSelector(state).history;
+export const visibleItemsCountSelector = (state) => domainSelector(state).visibleItemsCount;
+export const loadingSelector = (state) => domainSelector(state).loading;
+export const historyItemsSelector = (state) => {
```

Comment:
It's better to memoize this selector to avoid unnecessary rerenders.
```
import {createSelector} from 'reselect';
export const historyItemsSelector = createSelector(itemsSelector, (testItems) => {
 // ...
});
```